src/explorer.ts

```diff
@@ -333,12 +333,16 @@ class KubernetesSelectorResource extends KubernetesResource {
         if (!this.selector) {
             return [];
         }
-        const selectors = [];
-        for (const sel in this.selector.matchLabels) {
-            selectors.push(`${sel}=${this.selector.matchLabels[sel]}`);
+        let selectorString = "";
+        if (this.selector.matchLabels) {
+            const selectors = [];
+            selectors.push("-l");
+            for (const sel in this.selector.matchLabels) {
+                selectors.push(`${sel}=${this.selector.matchLabels[sel]}`);
+            }
+            selectorString = selectors.join(",");
```

Comment:
This looks like it will result in `-l,foo=bar" (with a comma) instead of `-l foo=bar`.  Can you confirm it's correct still correct if there are matchLabels present?

Reply:
fixed this.